book.rb

```diff
@@ -0,0 +1,9 @@
+# Represents the books class
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+  end
+end
```

Comment:
- Hey, let's add also the method for the rentals for every book rented by the person since the rentals in an intermediary between a person and a book to establish the has-many relationship.

```ruby
def add_rental(person, date)
    Rental.new(date, person, self)
  end
```
- The date is included to keep track of the date each book is rented and also, the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association` which is why the self is added in the parameter.